org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/filebased/ConfigurationBasedNamespaceManager.java

```diff
@@ -32,14 +32,14 @@ public ConfigurationBasedNamespaceManager(Configuration configuration) {
 		// globally set prefixes
 
 		// if that behavior is not desired, the code has to be moved to "generatePrefix" which checks for existence, ...
-		this.configuration.setProperty("http://www.w3.org/2001/XMLSchema", "xsd");
-		this.configuration.setProperty("http://www.w3.org/XML/1998/namespace", "xmlns");
+		this.configuration.setProperty(Namespaces.W3C_SCHEMA, "xsd");
+		this.configuration.setProperty(Namespaces.W3C_NAMESPACE, "xmlns");
 		this.configuration.setProperty(Namespaces.TOSCA_NAMESPACE, "tosca");
 		this.configuration.setProperty(Namespaces.TOSCA_WINERY_EXTENSIONS_NAMESPACE, "winery");
 
 		// example namespaces opened for users to create new types
-		this.configuration.setProperty("http://www.example.org", "ex");
-		this.configuration.setProperty("http://www.opentosca.org/nodetypes", "otnt");
+		this.configuration.setProperty(Namespaces.EXAMPLE_NAMESPACE, "ex");
```

Comment:
Here, it is OK to have a different block, because these namespaces are not standardized ones.